recipes/cdm/build.sh

```diff
@@ -0,0 +1,8 @@
+mkdir -p _build && cd _build
+
+cmake ../src \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_BUILD_TYPE=Release
+
```

Comment:
Is there a `make check/test` we can run here?

Reply:
There isn't yet, unfortunately. I'm working with the main author to put some better tests together, though.

(A grad student is waiting for a binary version of this package, which is why I've put this together before adding tests. I hope that's ok.)